modulemd/v2/modulemd-module-stream.c

```diff
@@ -990,3 +990,43 @@ modulemd_module_stream_build_depends_on_stream (ModulemdModuleStream *self,
 
   return klass->build_depends_on_stream (self, module_name, stream_name);
 }
+
+    
+static gboolean
+modulemd_module_stream_default_equals (ModulemdModuleStream *self, ModulemdModuleStream *other)
+{
+  g_return_val_if_fail (MODULEMD_IS_MODULE_STREAM (self), FALSE);  
```

Comment:
Test that both inputs are of the correct object type.

```suggestion
  g_return_val_if_fail (MODULEMD_IS_MODULE_STREAM (self), FALSE);
  g_return_val_if_fail (MODULEMD_IS_MODULE_STREAM (other), FALSE);
```